_install-and-configure/additional-plugins/index.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Additional plugins
+parent: Installing plugins
+nav_order: 10
+
+---
+
+# Additional plugins
+
+There are many more plugins available on top of what is provided by the standard distribution of OpenSearch. These additional plugins have been built by OpenSearch developers or members of the OpenSearch community. While it isn't possible to provide an exhaustive list (since many plugins are not maintained in an OpenSearch GitHub repository), these are some of the plugins that can be installed using one of the installation options, for example, using the command `bin/opensearch-plugin install <plugin-name>`.
+
```

Comment:
You'll find the following plugins, as well as others in the [OpenSearch/plugins](https://github.com/opensearch-project/OpenSearch/tree/main/plugins) directory in GitHub.